src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -73,6 +73,16 @@ public static void VerifyGraph(Compilation compilation, string expectedFlowGraph
         {
             var actualFlowGraph = GetFlowGraph(compilation, graph);
             OperationTreeVerifier.Verify(expectedFlowGraph, actualFlowGraph);
+
+            // Basic block reachability analysis verification using a test-only dataflow analyzer
```

Comment:
This will test the new Workspace layer dataflow walker for every CFG generated in compiler's flow analysis tests.